modules/red-hat-quay-namespace-auto-pruning-overview.adoc

```diff
@@ -2,7 +2,7 @@
 [id="red-hat-quay-namespace-auto-pruning-overview"]
 = {productname} auto-pruning overview
 
-{productname} administrators can set up auto-pruning policies on namespaces and repositories. This feature allows for image tags to be automatically deleted within a namespace or a repository based on specified criteria, which allows {productname} organization owners to stay below the storage quota by automatically pruning content.
+{productname} administrators can set up auto-pruning policies on organizations and repositories; administrators can also set up auto-pruning policies at the registry level so that they apply to all organizations, or all newly created organizations. This feature allows for image tags to be automatically deleted within an organization or a repository based on specified criteria, which allows {productname} organization owners to stay below the storage quota by automatically pruning content.
```

Comment:
should `, or all newly created organizations` be changed to `, including all newly created organizations`?